scripts/README.md

```diff
@@ -0,0 +1,94 @@
+# Updating tormays -material to environemnts
+
+General overview:
+
+These instructions will guide through various steps. Actual steps
+taken is dependent on the task at hand.
+
+- upload generated polygons to Haitaton file share (for others to access)
+- download polygons from file share to local disk (if needed)
+- update GIS materials from local disk to local dev database
+- update to dev
+- update to test
+
+Support for staging and production environments can be easily added later on, if needed.
+
+Prerequisites:
+
+- tormays polygon gis material is already generated
+
+## Prerequisite: Generate tormays polygons
+
+Generating tormays-polygon data is instructed in `gis-material-update/README.md`
+
+## Upload generated polygons to Haitaton file share
+
+Script run will resolve fileshare name and detect processed materials from
+processing output directory (`gis-material-update/haitaton-gis-output`).
+
+Commands are printed to stdout during script run, if processed material is found from directory.
+
+Following steps will provide commands to run:
+
+- ensure VPN is active and you are logged to OpenShift
+- select correct project, as instructed in oc login `oc project <projectname>`
+- run `sh hel-gis-print-upload-commands.sh`
+- copy and paste selected commands to run
```

Comment:
Why doesn't the script run these commands?

Reply:
Tällä saa ajettua kaikki komennot. Päivitän dokumentaatioon.
`sh hel-gis-print-upload-commands.sh | sh`